cmd/lifecycle/analyzer.go

```diff
@@ -60,6 +61,9 @@ func (a *analyzeCmd) Args(nargs int, args []string) error {
 	if a.cacheImageTag == "" && a.cacheDir == "" {
 		cmd.DefaultLogger.Warn("Not restoring cached layer metadata, no cache flag specified.")
 	}
+	cmd.UpdateAnalyzedPath(&a.analyzedPath, a.analyzeArgs.platformAPI, a.layersDir)
+	cmd.UpdateGroupPath(&a.groupPath, a.analyzeArgs.platformAPI, a.layersDir)
```

Comment:
```suggestion
	cmd.UpdateAnalyzedPath(&a.analyzedPath, a.platformAPI, a.layersDir)
	cmd.UpdateGroupPath(&a.groupPath, a.platformAPI, a.layersDir)
```
Since `analyzeArgs` is embedded we can reference `a.platformAPI` just like we reference `a.layersDir`.